### PR TITLE
Cleanup dead feature

### PR DESCRIPTION
**Pull request by iMoses**
Simplify interface + reduce coupling

### CODE REVIEW

File: packages/ag-charts-community/src/chart/chart.ts [hunk 1]
```diff
@@ -256,7 +256,7 @@ export abstract class Chart extends Observable {
         const container = resources?.container;
 
         const root = new Group({ name: 'root' });
-        const titleGroup = new Group({ name: 'titles', layer: true, zIndex: Layers.SERIES_LABEL_ZINDEX });
```
Comment (alantreadway) [inline packages/ag-charts-community/src/chart/chart.ts]: Can we keep this, it's useful for debugging?

File: packages/ag-charts-enterprise/src/features/status-bar/statusBar.ts [hunk 2]
```diff
@@ -252,7 +252,7 @@ export class StatusBar
         this.labelGroup.visible = false;
 
         this.destroyFns.push(
-            ctx.scene.attachNode(this.labelGroup, 'titles'),
+            ctx.scene.attachNode(this.labelGroup),
```
Comment (alantreadway) [inline packages/ag-charts-enterprise/src/features/status-bar/statusBar.ts:260]: This was hacked in to fix https://ag-grid.atlassian.net/browse/CRT-394 - can you verify that this case still works after this change?

Comment (iMoses) [reply packages/ag-charts-enterprise/src/features/status-bar/statusBar.ts:260]: Verified and works as expected

**Review by alantreadway — APPROVED** [review]

**Pull request merged**